src/components/Fundraiser.tsx

```diff
@@ -29,18 +29,28 @@ interface FundraiserState extends JSONObject {
   goal: number | null;
   nonprofitInfo: EveryNonprofitInfo | null;
   supporters: number;
+  status: FundraiserStatus | 'unknown';
 }
 
-function generateFundraiserURL(
+function generateBaseFundraiserURL(
+    fundraiserInfo: SerializedEveryExistingFundraiserInfo | null,
+    nonprofitInfo: EveryNonprofitInfo | null
+): string {
+    if (!fundraiserInfo || !nonprofitInfo) return '';
+    return `https://every.org/${nonprofitInfo.primarySlug}/f/${fundraiserInfo.slug}`;
+}
+
+function generateDonateFundraiserURL(
     fundraiserInfo: SerializedEveryExistingFundraiserInfo | null,
     nonprofitInfo: EveryNonprofitInfo | null,
     subreddit: string | undefined
 ): string {
-  if (!fundraiserInfo) return ''; // TODO: better default?
-  const utm_content = fundraiserInfo.id ? `&utm_content=${fundraiserInfo.id}` : '';
-  const utm_campaign = subreddit ? `&utm_campaign=${subreddit}`: '';
-  const utm = `?utm_source=reddit&utm_medium=fundraisers${utm_content}${utm_campaign}`;
-  return `https://every.org/${nonprofitInfo?.primarySlug}/f/${fundraiserInfo.slug}#/donate${utm}`;
+    const baseUrl = generateBaseFundraiserURL(fundraiserInfo, nonprofitInfo);
+    if (!baseUrl) return ''; // TODO: better default?
```

Comment:
We should figure out what this looks like :/

Reply:
From a UI perspective?

Reply:
from a UX yes. if the baseURL can't be populated what do we want to default to?

Reply:
I think the fundraiser post is a fail if we cant generate that--we should probably cancel the post creation/delete the post